src/EditorFeatures/Core/Implementation/EditAndContinue/ActiveStatementTrackingService.cs

```diff
@@ -139,9 +139,11 @@ public void EndTracking()
             }
 
             private void DocumentOpened(object sender, DocumentEventArgs e)
-                => DocumentOpenedAsync(e.Document);
+            {
+                _ = DocumentOpenedAsync(e.Document);
+            }
 
-            private async void DocumentOpenedAsync(Document document)
+            private async Task DocumentOpenedAsync(Document document)
```

Comment:
same comment as above.  being Task returning feels highly misleading and error prone.  We've had people do this and not then handle cancellation/exceptions properly.  'async void' is a nice clear marker of "this is fire and forget, just async, no one will await, so you have to handle these cases yourself".